mbuild/conversion.py

```diff
@@ -59,22 +61,34 @@ def load(filename_or_object,
     rigid : bool, optional, default=False
         Treat the compound as a rigid body
     backend : str, optional, default=None
-        Backend used to load structure from file. If not specified, a default
-        backend (extension specific) will be used.
+        Backend used to load structure from file or string. If not specified,
+        a default backend (extension specific) will be used.
     smiles: bool, optional, default=False
-        Use Open Babel to parse filename as a SMILES string
+        Use RDKit or OpenBabel to parse filename as a SMILES string
         or file containing a SMILES string.
+    smiles_seed: int, optional, default=-1
```

Comment:
Is there a reason why this `smiles_seed` is to be allowed as an explicit keyword argument rather than a part of `kwargs`?

Reply:
I agree this should be part of kwargs

Reply:
Good point, will make that change